toolkits/global/packages/global-buttons/scss/50-components/buttons.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * Regular buttons
+ *
+ */
+
+.c-button {
+	@include button($button--default);
+}
+
+.c-button--small {
+	font-size: $button--font-size-small;
+	line-height: $button--line-height-small;
+}
+
+.c-button--large {
+	font-size: $button--font-size-large;
+	line-height: $button--line-height-large;
+}
+
+.c-button--primary {
+	@include button-theme($button--primary);
+}
+
+.c-button--inline {
+	display: inline-block;
```

Comment:
I think inline-block makes sense as the default for buttons?
That's how the behave out of the box in the browser.
Then `c-button--block`? 😬